kpi/templates/registration/activation_email.txt

```diff
@@ -1,13 +1,15 @@
 {% load i18n %}
-{% trans "Thanks for signing up with KoBoToolbox!" %}
+{% trans "Dear" %} {{ user }},
 
-{% trans "Confirming your account will give you access to KoBoToolbox applications. Please visit the following URL to finish activation of your new account." %}
+{% trans "Thank you for signing up with KoBoToolbox!" %}
 
-{{ kpi_protocol }}://{{ site.domain }}/accounts/activate/{{activation_key}}
+{% trans "Confirming your account will give you full access to all KoBoToolbox applications. Please visit the following URL to finish the activation of your new account:" %}
 
-{% trans "Your username is: " %} {{ user }} 
+{{ kpi_protocol }}://{{ site.domain }}/accounts/activate/{{activation_key}}/
 
-{% trans "Please visit https://community.kobotoolbox.org/ to find information on how to get started. There you can also post questions to the community (recommended) or to us directly." %}
+{% trans "Your username is:" %} {{ user }}
+
+{% trans "Please visit our help pages https://support.kobotoolbox.org to find information on how to get started and visit our user forum at https://community.kobotoolbox.org in case you have any questions or difficulties" %}
```

Comment:
Now that we have URLs for community and support in the configuration, we should use one of those here. I guess it should be `SUPPORT_URL`?
https://github.com/kobotoolbox/kpi/blob/1a486956852a8de442132c6e502bf9329c801351/kobo/settings/base.py#L145-L154

Reply:
So in this case it could be 
```
{% trans "Please visit our help pages at {{ KOBO_SUPPORT_URL }} to find information on how to get started and visit our user forum at {{ KOBO_COMMUNITY_URL }} in case you have any questions or difficulties." %}
```

Reply:
Almost :wink: We need to change something in the Python code to pass that configuration information into the email template: https://django-constance.readthedocs.io/en/latest/#django-templates